.github/workflows/main.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         toolchain: ${{ matrix.rust }}
         override: true
 
-    - name: Build without default features
-      run: cargo build --verbose --no-default-features
+    - name: Build with minimal features (no_std)
+      run: cargo build --verbose --no-default-features --features libm
```

Comment:
How does it work without a specific `libm` feature flag?

Reply:
There's an error that gets emitted if neither `libm` nor `std` is used:
https://github.com/RazrFalcon/tiny-skia/pull/28/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R32

Reply:
Oh, sorry, I formulated my question incorrectly. Why does it works fine while it's not defined in the Cargo.toml `[features]` section?

Reply:
Oh, every optional crate is also a feature. Not sure why they thought this is a good idea. If you want I can make it a named feature in the features section instead. (Not sure about the name then though).

Reply:
I see. I didn't know that.